src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.uwp.cs

```diff
@@ -77,14 +79,12 @@ void OnInkPresenterStrokesCollected(InkPresenter sender, InkStrokesCollectedEven
 			foreach (var point in points)
 				elementPoints.Add(point);
 
-			if (elementPoints.Count > 0)
-			{
-				if (Element.DrawingCompletedCommand?.CanExecute(null) ?? false)
-					Element.DrawingCompletedCommand.Execute(elementPoints);
-			}
+			Element.OnDrawingCompleted();
 
 			if (Element.ClearOnFinish)
```

Comment:
maybe it should be removed

Reply:
What should be removed?

Reply:
If Element.ClearOnFinish is executed in DrawingView.shared.cs

Reply:
The `Clear` function does some tidy up logic in the renderer so I think it needs to stay.